release-content/0.15/release-notes/14212_Component_Lifecycle_Hook__Observer_Trigger_for_replaced_va.md

```diff
@@ -1,4 +1,79 @@
 <!-- Component Lifecycle Hook & Observer Trigger for replaced values -->
 <!-- https://github.com/bevyengine/bevy/pull/14212 -->
 
-<!-- TODO -->
+Bevy 0.14 introduced Component Lifecycle Hooks and Observers, and included several built-in observer triggers for each way that components could be added to or removed from entities. However, there was a hole in this API:
```

Comment:
This example is really neat, but we should have a sentence explaining that we're missing an entry in the 2x2 grid before diving into it.